blog-server-api/src/endpoints/author_subscribe/handler.rs

```diff
@@ -35,12 +38,17 @@ async fn http_handler(
         return Err(Forbidden);
     }
 
-    if author.base.telegram_id.is_none() {
+    if let Some(telegram_id) = author.base.telegram_id {
+        let event = SubscriptionStateChanged {
+            blog_user_id: id,
+            user_telegram_id: telegram_id,
+            new_state: subscribe,
+        };
+        event_bus_service.publish(event).await;
```

Comment:
Should we await on different queue?

Reply:
yep, now we spawn this one with async move